### ISSUE TITLE
ep count does not change in extension when i click the + sign in any anime

### ISSUE THREAD

**Issue by PerfectingBad** [TehNut/AniMouto#74]
After the new UI in the extension change, I can't update the episode count in the extension. Before it was instant when I hit the + sign but now no matter how many times I click that + sign ep count doesn't change  idk in general it changes in the database of mine but in that extension, the ep number does not change.
I tried updating the extension, but it didn't work. I thought maybe its new and in edge it doesn't implemented properly but I checked in chrome also but in chrome also it didn't change

edit: it turns out that the ep count does changes but it doesn't show the increased ep number. And here a catch, it only shows/ or seen by the user after I click the new reload button at top right.
Why its that. Why do I have to click the reload sign every time i click the + in any anime selected. Is this the feature or bug.If this is the new feature then pls revert back it

**Comment by KiriCattus** [comment]
It doesn't update until you either reload the list of anime or minimise then open up the extension window again, however I did notice this while testing to see if over-progression prevention was still a feature as I used that protection to prevent me miss clicking or not remembering I had already clicked when it had previously not updated right away in 2.* but it seem over-progression protection has been removed entirely now from the settings tab and extension in 3.*

In short the value doesn't increase when you click + on any anime until you reload the tab or re-enter the extension and over-progression protection is no longer there.

Other than this small issue I'm loving the new update TehNut, thanks :3

**Comment by PerfectingBad** [comment]
just imagine, if u wana update a bunch of episodes at once,  won't it  be so much annoying to click + then reload, then click + then reload, and over and over and over again . I hate this change.

**Comment by TehNut** [comment]
It's a bug, calm down.

**Comment by ereinstein** [comment]
There hasn't been a new release but this seems to be fixed on my end. It never worked a few days ago but today it just worked.

**Comment by TehNut** [comment]
Honestly I think this was just related to the ongoing server problems. It seemed very inconsistent to me and it worked fine up until release, which is about when the server issues started.

...maybe the server stuff is my fault 🙃

**Comment by TehNut** [comment]
> ***ongoing*** server problems

**Comment by Invisible40** [comment]
Still getting this error btw

**Comment by Elryyyyyyyy** [comment]
> It's a bug, calm down.

 Yeah you should maybe stop calming down and work on that Bug, its been 4 months.
Fix this, its annoying !

**Comment by Elryyyyyyyy** [comment]
@TehNut

**Comment by TehNut** [comment]
I work on things when I have time, energy, and motivation. Acting like you are right now actively detracts from 2 of those :)

**Comment by Elryyyyyyyy** [comment]
> I work on things when I have time, energy, and motivation. Acting like you are right now actively detracts from 2 of those.

Maybe you should just find another dev to take over by now, you started to migrate to v3 rewrite but only like halfway, just finish it ffs.

Your cheap excuses are funny its been 4 months since this is reported.. Im pretty sure you had time or energy for it in 4 months, if you lack motivation that much i feel bad for your life.
